runtime/lua/vim/treesitter/_fold.lua

```diff
@@ -5,35 +5,20 @@ local Range = require('vim.treesitter._range')
 local api = vim.api
 
 ---@class TS.FoldInfo
----@field levels table<integer,string>
----@field levels0 table<integer,integer>
----@field private start_counts table<integer,integer>
----@field private stop_counts table<integer,integer>
+---@field levels string[] the foldexpr value for each line
```

Comment:
This is a bit ambiguous

```suggestion
---@field levels string[] the foldexpr result for each line
```